src/uct/ib/rc/base/rc_ep.h

```diff
@@ -163,6 +172,12 @@ struct uct_rc_ep {
     uct_rc_fc_t         fc;
 };
 
+typedef struct uct_rc_fc_request {
+    uct_pending_req_t req;
+    uct_ep_t          *ep;
+    char              priv[UCT_RC_FC_TL_PRIV_LEN];
```

Comment:
in UCT API, we used "priv" to avoid exposing arbiter type. However the usage here is internal. Can we just put ucs_arbiter_elem_t instead of priv?